adhocracy-plus/assets/scss/components/_modal.scss

```diff
@@ -0,0 +1,48 @@
+// overwriting bootstrap 'modal' classes
+.modal-header {
+    h2 {
+        font-family: $font-family-base;
+        font-size: $font-size-lg;
+    }
+}
+
+.modal-footer {
+    button {
+        margin-top: 0;
+        margin-bottom: 0;
+    }
+
+    button.submit-button {
+        @extend .btn;
+        @include button-bg($print-black, $print-black, $body-bg, 15%);
+        margin-right: 0;
+        order: 2;
```

Comment:
we already have styling for these buttons and we want them to be consistent so I would not add special styles for the modals, instead maybe move the existing selectors to buttons.scss and extend the button styles we will use, setting our own will mean we might get inconsistencies in our button styles again, this is something it would be great to clean up so we mostly just want to extend our existing styles

Reply:
this modal specific classes are coming from mB and I think it makes more sense, because otherwise we would tie both projects to the same class names which could lead to problems for instance in mB `btn btn--primary` and here `btn btn--light`.

I think we are more flexible to extend these modal button classes with the specific ones needed in each project.

Here I just didn't manage to `extend from two classes`, that is not ideal I agree...Not sure how to extend it better, rather having copied the styles (in this case `@include button(....`)

Reply:
submit-button in a bootstrap button I think but we don't want to have multiple different submit buttons, no matter where they are as they will hopefully do the same thing so we should use our standard classes btn and btn--default or whichever suits the design and as they are buttons they can be moved to button.scss and also we can not double up with them in shame.scss too